homeassistant/components/iskra/entity.py

```diff
@@ -0,0 +1,39 @@
+"""Base entity for Iskra devices."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER
+from .coordinator import IskraDataUpdateCoordinator
+
+
+class IskraEntity(CoordinatorEntity[IskraDataUpdateCoordinator]):
+    """Representation a base Iskra device."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, coordinator: IskraDataUpdateCoordinator) -> None:
+        """Initialize the Iskra device."""
+        super().__init__(coordinator)
+        device = coordinator.device
```

Comment:
and set this to `self.device`, you will probably use that in other platforms as well